documentation/modules/upgrading-mtv-ui.adoc

```diff
@@ -32,10 +32,10 @@ endif::[]
 +
 [source,terminal,subs=attributes+]
 ----
-$ {oc} get pod -n openshift-marketplace | grep <catalog_source>
+$ {oc} get pod -n openshift-marketplace | grep <catalog_source> <1>
 ----
+<1> Specify the catalog source, for example, redhat-operators.
```

Comment:
"redhat-operators" should be marked up with backticks because it is entered on the CLI.